packages/Cmfcmf-React-Godot.package/CMFReactNodeGodot.class/instance/asCMFReactNode.st

```diff
@@ -0,0 +1,4 @@
+converting
+asCMFReactNode
+
+	
```

Comment:
Should this return `self`?

Reply:
yep SB removes these because the parser adds these when they are missing, so SB takes a guess to always remove them... should fix this -.-